metomi/rosie/usertools/ldaptool.py

```diff
@@ -24,7 +24,7 @@
 except ImportError:
```

Comment:
Do we need `ldap` as an optional dependency in `setup.py`?

Reply:
Yes, and it looks like it ought to be ldap3 rather than ldap. Will fix.